webpack.common.babel.js

```diff
@@ -16,6 +16,7 @@ export default {
   output: {
     path: resolve(__dirname, "dist"),
     publicPath: "/",
+    libraryExport: "default",
```

Comment:
positive affirmation:
+ Gosh, i struggled SO long trying to figure out how to get the default export to work. Webpack can be so insane that tracking down these options is a challenge. Thanks for this!